src/plug/Views/TimeLimitView.vala

```diff
@@ -169,19 +187,46 @@ namespace PC.Widgets {
 
             column_spacing = 12;
             row_spacing = 6;
+
+            ///TRANSLATORS: First %s is the user's name, second is the sentence-case for "weekday" or "weekend"
+            var message_not_limited = _("Screen Time for %s will not be limited during this period on %s.").printf (
+                user.get_real_name (),
+                sentence_case
+            );
+
+            ///TRANSLATORS: First %s is the user's name, second is the sentence-case for "weekday" or "weekend"
+            var message_limited = _("%s will only be able to log in during this time on %s, and will be automatically logged out once this period ends:").printf (
+                user.get_real_name (),
+                sentence_case
+            );
+
+            var limit_description = new Gtk.Label (message_not_limited);
+            limit_description.wrap = true;
+            limit_description.xalign = 0;
+
+
             attach (label, 0, 0);
             attach (enable_switch, 1, 0);
-            attach (from_label, 0, 1);
-            attach (picker_from, 1, 1);
-            attach (to_label, 2, 1);
-            attach (picker_to, 3, 1);
+            attach (limit_description, 0, 1, 4, 1);
+            attach (from_label, 0, 2);
+            attach (picker_from, 1, 2);
+            attach (to_label, 2, 2);
+            attach (picker_to, 3, 2);
 
             bind_property ("active", enable_switch, "active", GLib.BindingFlags.BIDIRECTIONAL);
             bind_property ("active", from_label, "sensitive", GLib.BindingFlags.SYNC_CREATE);
             bind_property ("active", picker_from, "sensitive", GLib.BindingFlags.SYNC_CREATE);
             bind_property ("active", to_label, "sensitive", GLib.BindingFlags.SYNC_CREATE);
             bind_property ("active", picker_to, "sensitive", GLib.BindingFlags.SYNC_CREATE);
 
+            notify["active"].connect (() => {
+                if (active) {
+                    limit_description.set_text (message_limited);
+                } else {
+                    limit_description.set_text (message_not_limited);
+                }
```

Comment:
```suggestion
                if (active) {
                    limit_description.text = message_limited;
                } else {
                    limit_description.text = message_not_limited;
                }
```

We prefer setting properties directly instead of using `set` methods